src/main/java/org/jabref/gui/groups/GroupTreeViewModel.java

```diff
@@ -260,9 +260,60 @@ boolean onlyMinorChanges(AbstractGroup oldGroup, AbstractGroup newGroup) {
         return true;
     }
 
+    /**
+     * Opens "Rename Group Dialog" and change name of group
+     */
+
+    public void renameGroup(GroupNodeViewModel oldGroup) {
+        currentDatabase.ifPresent(database -> {
+            AbstractGroup oldGroupDef = oldGroup.getGroupNode().getGroup();
+            String oldGroupName = oldGroupDef.getName(); // Zachytenie starého názvu pred otvorením dialógu
```

Comment:
Please translate the comment (or remove it)

Reply:
removed